docs/install/setting_up_cluster_on_amazon_web_services.md

```diff
@@ -291,12 +299,13 @@ When you start a cluster, it uses the existing EC2 instances associated with the
 ./snappy-ec2 -k my-ec2-key -i ~/my-ec2-key.pem start cluster-name
 ```
 !!! Note
-	The start command (or launch command with `--resume` option) ignores the `--locators`, `--leads`, or `--stores` options, and launches the SnappyData cluster on existing instances. The configuration files however, are read if they are present in the location mentioned above. You therefore must ensure that every time you use the `start` command, the number of entries in configuration files are equal to the number of instances in their respective security group.
+	The `start` command (or `launch` command with `--resume` option) ignores the `--locators`, `--leads`, or `--stores` options, and launches the SnappyData cluster on existing instances.
+	The configuration options, however, are read if they are provided, overriding those provided when the cluster was launched/started previously.
```

Comment:
However, the configuration options are read, if they are provided, by overriding those that are provided when the cluster was launched or started previously.